tests/fixtures/lxd-test.ts

```diff
@@ -0,0 +1,26 @@
+import { TestInfo, test as base } from "@playwright/test";
+
+export type LxdVersions = "5.0-stable" | "latest-edge";
+const LXD_CHANNEL = process.env.LXD_CHANNEL as LxdVersions;
```

Comment:
Shall we have a fallback for running locally? I guess this would currently just fail, if `LXD_CHANNEL` is not set in the environment.

Reply:
Edit: Not needed if we have the combined project name.